packages/mdc-button/mdc-button.scss

```diff
@@ -22,6 +22,39 @@
 @import "@material/typography/mixins";
 
 // postcss-bem-linter: define button
+
+.mdc-button {
+  @include mdc-ripple-base;
+  @include mdc-ripple-bg((pseudo: "::before"));
+  @include mdc-ripple-fg((pseudo: "::after"));
+
+  overflow: hidden;
+
+  @include mdc-theme-dark(".mdc-button", true) {
```

Comment:
I think this can just be `@include mdc-theme-dark {` here. We use the first argument in the case where we need to style an _BEM element_ class  to adhere to a dark theme treatment

```scss
.mdc-component__element {
  @include mdc-theme-dark(".mdc-component") {
    // ...
  } 
}
```

And we usually use the second argument when we're styling a modifier class for dark theme treatment, e.g.

```scss
.mdc-component--mod {
  @include mdc-theme-dark(".mdc-component", true) {
    // ...
  }
}
```

The [mdc-theme README](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme#mdc-theme-dark-mixin) has more info on this.